src/main/java/frc/robot/subsystems/intake/Command/Feed.java

```diff
@@ -1,15 +1,18 @@
 package frc.robot.subsystems.intake.Command;
 
 import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.intake.BeamBreaker;
 import frc.robot.subsystems.intake.Intake;
 
 public class Feed extends CommandBase {
     private final Intake intake;
+    private final BeamBreaker beamBreaker;
     private final double power;
     private final double angle;
 
     public Feed(double power, double angle) {
         this.intake = Intake.getInstance();
+        this.beamBreaker = BeamBreaker.getInstance();
```

Comment:
Subsystems should be passed as parameters to commands.

Reply:
done.

Reply:
What about the arm?
Also, don't resolve conversations by yourself – it's the reviewer's role, so they can see the fix actually fixed the problem (which didn't fully happen here).